src/test/java/seedu/address/testutil/TypicalPets.java

```diff
@@ -24,41 +27,41 @@
  */
 public class TypicalPets {
 
-    public static final Pet COCO = new PetBuilder().withName("Coco")
+    public static final Pet AMY = new PetBuilder().withName("Amy")
             .withSpecies("Dog").withDateOfBirth("1/6/2015")
-            .withGender(Gender.FEMALE).withFoodList("Brand A")
+            .withGender(Gender.FEMALE).withFoodList("Brand A:10")
             .withTags("darkFur").build();
-    public static final Pet GARFIELD = new PetBuilder().withName("Garfield Arbuckle")
-            .withSpecies("Cat").withFoodList("Brand B")
+    public static final Pet BOB = new PetBuilder().withName("Bob")
+            .withSpecies("Cat").withFoodList("Brand B:10")
             .withDateOfBirth("19/6/1978").withGender(Gender.MALE)
             .withTags("fat", "lazy").build();
```

Comment:
These two should be different from the two used in many other places otherwise tests will fail.